tests/test_ilastik/test_applets/edgeTraining/testEdgeTrainingSerializer.py

```diff
@@ -0,0 +1,134 @@
+import h5py
+import numpy
+import pandas
+import pytest
+
+from ilastikrag import Rag
+from ilastik.applets.edgeTraining import OpEdgeTraining
+from lazyflow.utility.testing import build_multi_output_mock_op, SlotDescription
+from ilastik.applets.edgeTraining.edgeTrainingSerializer import EdgeTrainingSerializer
+import vigra
+
+
+@pytest.fixture
+def empty_project_file():
+    with h5py.File("test_project1", mode="a", driver="core", backing_store=False) as test_project:
```

Comment:
Probably not important, but add a file extension just in case:
```suggestion
    with h5py.File("test_project1.h5", mode="a", driver="core", backing_store=False) as test_project:
```

Reply:
you mean because it's accessed in the serializer? Otherwise the file does not exist - it's in memory only. But yea... in any case.